src/decorator/string/IsStrongPassword.ts

```diff
@@ -0,0 +1,64 @@
+import { ValidationOptions } from '../ValidationOptions';
+import { buildMessage, ValidateBy } from '../common/ValidateBy';
+import isStrongPasswordValidator from 'validator/lib/isStrongPassword';
+
+export const IS_STRONG_PASSWORD = 'isStrongPassword';
+
+/**
+ * Options to be passed to IsStrongPassword decorator.
+ */
+export interface IsStrongPasswordOptions {
```

Comment:
You don't need to re-add the interfaces, you can directly import it from `valdator.js`. See other decorators for an example.